modules/tinymce/src/core/test/ts/browser/DragDropOverridesTest.ts

```diff
@@ -257,6 +257,31 @@ describe('browser.tinymce.core.DragDropOverridesTest', () => {
       });
     });
 
+    it('TINY-9025: Dragging CEF element outside the editor should not cause scrolling', async () => {
+      const editor = hook.editor();
+      editor.setContent(`
+      <p>CEF can get dragged before this one</p>
+      <p class="hidden" style="height: 1500px"></p>
+      <p id="separator" style="height: 100px"></p>
+      <p contenteditable="false" style="height: 200px; background-color: black; color: white">Draggable CEF</p>
+    `);
+      window.scroll({
+        top: window.innerHeight + 2000
+      });
+      const target = UiFinder.findIn(TinyDom.body(editor), 'p:contains("Draggable CEF")').getOrDie();
+      const initialScrollY = window.scrollY;
+      editor.getWin().scroll({
+        top: editor.getWin().innerHeight
+      });
+      Mouse.mouseDown(target);
+      window.dispatchEvent(new MouseEvent('mousemove', {
```

Comment:
should be used `Mouse.mouseMoveTo` instead?

Reply:
No, `Mouse.mouseMoveTo` only works within the editor, not outside of it if I'm not mistaken 🤷‍♂️

Reply:
@hamza0867 you are incorrect there... anything Agar related can be used by anything. It's `mcagar` helpers that are editor specific.